### PR TITLE
[new release] melange-numeral (0.0.1)

### PR DESCRIPTION
**Pull request by jchavarri**
Melange bindings for Numeral

- Project page: <a href="https://github.com/ahrefs/melange-numeral/">https://github.com/ahrefs/melange-numeral/</a>

##### CHANGES:

Initial release.

Migrated to Melange v2.

### CODE REVIEW

File: packages/melange-numeral/melange-numeral.0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+tags: ["melange" "org:ahrefs"]
+homepage: "https://github.com/ahrefs/melange-numeral/"
+bug-reports: "https://github.com/ahrefs/melange-numeral/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.8"}
+  "melange" {>= "2.0.0"}
+  "reason" {>= "3.10.0"}
+  "opam-check-npm-deps" {with-test} # todo: use with-dev-setup once opam 2.2 is out
+  "ocaml-lsp-server" {with-test}
```
Comment (kit-ty-kate) [inline packages/melange-numeral/melange-numeral.0.0.1/opam:22]:
```suggestion
  "opam-check-npm-deps" {with-dev-setup}
  "ocaml-lsp-server" {with-dev-setup}
```

Comment (jchavarri) [reply packages/melange-numeral/melange-numeral.0.0.1/opam]: I prefer `with-dev-setup` :) but then is there a way to install add these packages as deps while still using opam 2.1? I am imagining a way to set variables to true dynamically from the command line?

**Review by haochenx — APPROVED** [review]
CI is failing at this moment on macos machines but it seems that they are all the CI's fault. I have restarted the jobs.

Yet since all other CI jobs succeeded and there is no reason to think macos will fail,
It looks to me that this PR is mergeable.

File: packages/melange-numeral/melange-numeral.0.0.1/opam (hunk 1 above)
Comment (kit-ty-kate) [reply packages/melange-numeral/melange-numeral.0.0.1/opam]: ah nevermind i now understand what you were saying. Sorry for my misunderstanding. Mmh, i guess the `opam var` incantation above is fine as a temporary solution although if it's just for yourself i would encourage you to use opam 2.2 directly instead.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**